grammars/go.cson

```diff
@@ -128,6 +128,10 @@
     'match': '\\b(iota|true|false|nil)\\b'
   }
   {
+    'match': '\\;'
+    'name': 'punctuation.terminator.statement.go'
```

Comment:
Sorry about violating the naming conventions. That is absolutely not OK.

I still think that the semicolon should be a delimiter, for the same reason as braces. So it shouldn't be part of https://github.com/atom/language-go/blob/master/grammars/go.cson#L162. Something similar to `meta.delimiter.terminator.semicolon.go`?